components/footer.tsx

```
@@ -1,18 +1,18 @@
 import Link from "next/link";
 
-export default function Footer({ disclaimer }: { disclaimer?: string }) {
+export default function Footer({
+  copyrightNotice,
+}: {
+  copyrightNotice?: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)